rust/src/program/execute.rs

```diff
@@ -148,7 +150,7 @@ impl<N: Network> ProgramManager<N> {
 mod tests {
     use super::*;
     use crate::{random_program_id, AleoAPIClient, RECORD_5_GATES};
-    use snarkvm_console::network::Testnet3;
+    use snarkvm::console::network::Testnet3;
```

Comment:
Currently we can't remove this, because even without default features, the SnarkVM crate imports lead the wasm unable to compile, and the snarkvm_wasm crate currently induces the broken parameter download path from the parameter crate.

Reply:
Also having it blows up the size of the `wasm` crate because it imports all the stuff in the SnarkVM base crate

Reply:
No worries, thanks